pom.xml

```diff
@@ -1375,7 +1376,8 @@
       <id>integration-tests</id>
       <activation>
         <property>
-          <name>integration-tests</name>
+          <name>deployArtifacts</name>
+          <value>true</value>
```

Comment:
deploy script set this prop so that integ tests modules can be included

Reply:
So, to build integ test bundle (after this patch), what should one execute?
previously it was 
```
mvn package -DskipTests -Dintegration-tests
```

Reply:
it's always used as `mvn package -Pintegration-tests`. this is just an activation property, which wasn't used.

Reply:
got it